src/main.rs

```diff
@@ -92,6 +92,15 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                     std::process::exit(exitcode::CONFIG);
                 }
             };
+
+            match pgcat::tls::reload_root_cert_store().await {
+                Ok(_) => (),
+                Err(_) => {
+                    // no need for the error! macro, because the reload_root_cert_store()
+                    // function already uses the error! macro.
+                    std::process::exit(exitcode::OSFILE);
```

Comment:
Not entirely sure you want to exit here, some setups may not want TLS or don't have the certs installed at all which could cause an error perhaps? I'm not entirely sure why the root store could give an error, but in any case, if TLS is optional, which it can be if we set `server_tls` to false, this should not stop the pooler from starting.

Reply:
Okay, I'll think about it.